zipline/algorithm.py

```diff
@@ -2234,6 +2235,23 @@ def set_max_leverage(self, max_leverage):
         control = MaxLeverage(max_leverage)
         self.register_account_control(control)
 
+    @api_method
+    def set_min_leverage(self, min_leverage, timedelta):
```

Comment:
I would still call this `deadline`.  `timedelta` is shadowing the name of a built-in type.

Reply:
What about `offset`, since `deadline` is what is computed and passed into `MinLeverage`

Reply:
As a consumer of this function, I don't think `offset` would be a very clear name for this parameter.  My immediate question would be "offset from what"?  Maybe `grace_period`?